src/share/vm/gc_implementation/g1/elasticHeap.cpp

```diff
@@ -166,7 +170,8 @@ void ElasticHeapConcThread::print_work_summary(uint uncommit_length, uint commit
     ShouldNotReachHere();
   }
   size_t bytes = list_length * HeapRegion::GrainBytes;
-  gclog_or_tty->print_cr("[Elastic Heap concurrent thread: %s " SIZE_FORMAT "%s spent %.3fs(concurrent workers: %u) ]",
+  gclog_or_tty->print_cr("[Elastic Heap%s: %s " SIZE_FORMAT "%s spent %.3fs(concurrent workers: %u) ]",
```

Comment:
You may want to add `PrintGCDetails && PrintElasticHeapDetails` in front of this line.

Reply:
ElasticHeapConcThread::print_work_summary is already guarded by options.